Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/BuildTopologyDictionary.h

```diff
@@ -68,15 +67,16 @@ class BuildTopologyDictionary
   friend std::ostream& operator<<(std::ostream& os, const BuildTopologyDictionary& BD);
   void printDictionary(std::string fname);
   void printDictionaryBinary(std::string fname);
+  void saveDictionaryRoot(const char* filename);
 
   int getTotClusters() const { return mTotClusters; }
   int getNotInGroups() const { return mNotInGroups; }
   int getNGroups() const { return mNumberOfGroups; }
 
  private:
   TopologyDictionary mDictionary; ///< Dictionary of topologies
-  std::map<unsigned long, std::pair<ClusterTopology, unsigned long>>
-    mTopologyMap; ///< Temporary map of type <hash,<topology,counts>>
+  std::unordered_map<unsigned long, std::pair<ClusterTopology, unsigned long>>
```

Comment:
In the previous version I've changed your std::unordered_map to std::map because otherwise it was failing the PR test on MacOS. With your reintroduction of the unordered_map the failure is back. Will it work with std::map for the moment, until we sort out the problem with unordered_map (apparently a root bug)?

Reply:
Thank you @shahor02, you just anticipated me. I passed to `unordered_map` since I thought that the problem was solved in the meanwhile. Then, when this probIem showed up, I suspected that the reason of the problem was `unordered_map`. However, the strange thing is that everything compiles on my laptop, i.e. on macOS.
For the time being I will pass to `map`